shared/desktop/app/main-window.desktop.tsx

```diff
@@ -276,6 +276,7 @@ export default () => {
     webPreferences: {
       backgroundThrottling: false,
       devTools: showDevTools,
+      enableRemoteModule: true,
```

Comment:
Seems this is needed to call Electron.remote. I don't know if it's needed on all 3 of the webPreferences

Reply:
yes it does need to be there